src/utils/colors-types.ts

```diff
@@ -0,0 +1,694 @@
+export interface Colors {
```

Comment:
getting these from primitives would be preferred, but since we process them here that doesn't seem practical